src/epics/timeline.js

```diff
@@ -0,0 +1,66 @@
+import { Observable } from 'rxjs';
+import { Map, OrderedMap } from 'immutable';
+import * as TimelineActions from 'app/actions/timeline';
+import RevisionEntity from 'app/entities/revision';
+
+const getWikiDomain = ( wiki, wikis ) => {
+	let wikiRecord = wikis.find( ( obj ) => {
+		return obj.get( 'id' ) === wiki;
+	} );
+
+	return wikiRecord.domain;
+};
+
+const buildUrl = ( store ) => {
```

Comment:
This function should take the query, not the store. That will make it easier to test.